packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -121,8 +172,91 @@ class ExpansionPanelList extends StatelessWidget {
   /// The duration of the expansion animation.
   final Duration animationDuration;
 
+  //Whether multiple panels can be open simultaneously
+  final bool _allowMultiplePanelsOpen;
+
+  @override
+  State<StatefulWidget> createState() => new _ExpansionPanelListState();
+}
+
+class _ExpansionPanelListState extends State<ExpansionPanelList> {
+  Map<int, bool> _openPanels = <int, bool>{};
+
+  @override
+  void initState() {
+    super.initState();
+    if (!widget._allowMultiplePanelsOpen) {
```

Comment:
Since you are always using the negation of this field, consider changing it to _preventMultiplePanelsOpen or something similar